COLLABORATORS.md

```diff
@@ -31,3 +30,4 @@
 - [yous](https://github.com/yous)
 - [marocchino](https://github.com/marocchino)
 - [MJayJang](https://github.com/MJayJang)
+- [yeqing](https://github.com/yeqing123)
```

Comment:
Left this file to be modified for maintainers

Reply:
ok